src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -102,6 +142,7 @@ public double getCurrentTime() {
 
   public double getVideoHieght() {
     if (playing) {
+      pause();
```

Comment:
Now this method name is not really describing what it is doing (pausing + getting video height)